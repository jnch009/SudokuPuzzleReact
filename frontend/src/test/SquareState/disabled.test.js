import { render, screen } from '@testing-library/react';
import SquareDisabledBtn from '../../Components/Square/squareState/Disabled';
import NoBorder from '../../Components/Square/squareBorders/None';

const num = 1;
const borderColor = 'blue';

it('Square Active', () => {
  const { container } = render(SquareDisabledBtn(NoBorder, num, borderColor));
  expect(container.firstChild).toMatchSnapshot();
  expect(container.firstChild.classList.contains('square')).toBeTruthy();
  expect(container.firstChild.classList.contains('primary')).toBeTruthy();
  expect(screen.getByText('1')).toBeTruthy();
});
