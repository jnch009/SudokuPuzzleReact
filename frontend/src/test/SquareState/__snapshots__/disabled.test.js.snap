// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Square Active 1`] = `
.c0 {
  color: #fff;
  cursor: not-allowed!important;
}

<button
  class="c0 square primary"
>
  1
</button>
`;
